apps/web/components/dashboard/bookmarks/BookmarkLayoutAdaptingCard.tsx

```diff
@@ -45,6 +47,41 @@ function BottomRow({
   );
 }
 
+function MultiBookmarkSelector({ bookmark }: { bookmark: ZBookmark }) {
+  const { selectedBookmarks, isBulkEditEnabled } = bulkActions();
+  const toggleBookmark = bulkActions((state) => state.toggleBookmark);
+  const [isSelected, setIsSelected] = useState(false);
+
+  useEffect(() => {
+    setIsSelected(selectedBookmarks.some((item) => item.id === bookmark.id));
+  }, [selectedBookmarks]);
+
+  if (!isBulkEditEnabled) return null;
+
+  return (
+    <button
+      className={cn(
+        "absolute left-0 top-0 z-50 h-full w-full bg-black bg-opacity-0",
+        {
+          "bg-opacity-10": isSelected,
+        },
+      )}
+      onClick={() => toggleBookmark(bookmark)}
+    >
+      <button className="absolute right-2 top-2 z-50 opacity-100">
+        <div
+          className={cn(
+            "flex h-4 w-4 items-center justify-center rounded-full border border-dashed border-gray-600",
+            isSelected ? "bg-black" : "bg-white bg-opacity-40",
```

Comment:
This effect is not as obvious in dark mode

Reply:
Thanks for testing in dark mode, fixed it.